Charon/filetypes/OpenPackagingConvention.py

```diff
@@ -0,0 +1,574 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# libCharon is released under the terms of the LGPLv3 or higher.
+
+from collections import OrderedDict  # To specify the aliases in order.
+from io import BytesIO, TextIOWrapper  # For the type of input of openStream and to create binary output streams for getting metadata.
+import json  # The metadata format.
+import re  # To find the path aliases.
+from typing import Any, Dict, List, IO, Optional
+import xml.etree.ElementTree as ET  # For writing XML manifest files.
+import zipfile
+
+from ..FileInterface import FileInterface  # The interface we're implementing.
```

Comment:
It's better to not use relative imports.

Reply:
This was already there (OpenPackagingConvention is just renamed from the old UltimakerFormatPackage file). I'm seeing a lot of other relative imports as well, maybe that's needed to make it work correctly in Cura or the embedded system (python path settings)?